docs/en/API-reference.md

```diff
@@ -129,10 +129,39 @@ dayjs(dayjs('2019-01-25')) // passing a Dayjs object to a constructor will also
 
 Returns a `boolean` indicating whether the `Dayjs`'s date is valid.
 
-```js
-dayjs().isValid()
-```
-
+- Browser
+  - Non-strict check.
+
+    Only checks if the value can be `instanceof Date`.
```

Comment:
to check if it could be parsed to a Date time, is it easy to understand?

Reply:
Yes, it is very clear. Just to get it straight, you'd prefer if I change this: `...value can be instanceof Date.` to this: `checks if value could be parsed to a Date time`?